netebpfext/net_ebpf_ext_xdp.c

```diff
@@ -420,15 +556,10 @@ net_ebpf_ext_layer_2_classify(
     net_xdp_ctx.ingress_ifindex =
         incoming_fixed_values->incomingValue[FWPS_FIELD_INBOUND_MAC_FRAME_NATIVE_INTERFACE_INDEX].value.uint32;
 
-    // TODO(issue #754): Support multiple clients and iterate through them.
-    // Also, the net_ebpf_extension_get_client_data() function is used because currently
-    // the structure is opaque except inside ebpf_ext_attach_provider.c.  However,
-    // this results in a slightly longer cycle count in the hot path to get to
-    // the client data here.   In the future, the client data field should be
-    // exposed in the .h file for us to access here.
```

Comment:
As far as I can tell the comments in 424-428 still apply and should be left in.

Reply:
I will analyze the generated retail code to see if the additional instructions are too many to keep this comment.